core/src/saros/filesystem/IContainer.java

```diff
@@ -34,4 +34,28 @@
   public IResource[] members() throws IOException;
 
   public String getDefaultCharset() throws IOException;
+
+  /**
+   * Returns a handle for the file with the given relative path to this resource.
+   *
+   * @param pathString a string representation of the path relative to this resource
+   * @return a handle for the file with the given relative path to this resource
+   * @throws NullPointerException if the given string is <code>null</code>
+   * @throws IllegalArgumentException if the given string represents an absolute path
+   */
+  public IFile getFile(String pathString);
+
+  public IFile getFile(IPath path);
+
+  /**
+   * Returns a handle for the folder with the given relative path to this resource.
+   *
+   * @param pathString a string representation of the path relative to this resource
+   * @return a handle for the folder with the given relative path to this resource
+   * @throws NullPointerException if the given string is <code>null</code>
+   * @throws IllegalArgumentException if the given string represents an absolute path
+   */
+  public IFolder getFolder(String pathString);
+
+  public IFolder getFolder(IPath path);
```

Comment:
JavaDoc

Reply:
I didn't think it was worth it to update the javadoc of the classes I haven't changed yet. When I am done with the rework (introducing "reference points" or whatever they will be called), the javadoc for all resource interfaces will have to be overhauled anyways. That is also the reason why I haven't removed the note that the functionality matches the eclipse methods if not mentioned otherwise.